<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-P2PQOR4" failures="1" tests="3" name="Submit Order Test" time="36.235" errors="0" timestamp="2024-07-23T19:41:13 IST">
  <testcase classname="rahulshettyacademy.tests.SubmitOrderTest" name="submitOrder" time="8.58"/>
  <testcase classname="rahulshettyacademy.tests.SubmitOrderTest" name="submitOrder" time="6.964"/>
  <testcase classname="rahulshettyacademy.tests.SubmitOrderTest" name="OrderHistoryTest" time="1.216">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at rahulshettyacademy.tests.SubmitOrderTest.OrderHistoryTest(SubmitOrderTest.java:49)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- OrderHistoryTest -->
</testsuite> <!-- Submit Order Test -->
